src/ox_modules/module-mob/commands/waitForOpen.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2015-present CloudBeat Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ */
+
+/**
+ * @summary Opens an URL.
```

Comment:
Add description why this command is needed over regular `open`.
And for `open` itself command add a remark about `waitForOpen`